apps/web-giddh/src/app/shared/create-address/create-address.component.ts

```diff
@@ -116,9 +116,17 @@ export class CreateAddressComponent implements OnInit, OnDestroy {
      *
      * @memberof CreateAddressComponent
      */
-    public checkGstNumValidation(): void {
-        if (this.addressForm.get('taxNumber').value && this.addressForm.get('taxNumber').valid && this.addressConfiguration.tax.name === 'GSTIN') {
-            this.getGstConfirmationPopup();
+    public checkGstNumValidation(element: HTMLInputElement): void {
+        if (this.addressForm.get('taxNumber').value && this.addressForm.get('taxNumber').valid) {
+            this.toasterService.clearAllToaster();
```

Comment:
```suggestion
            element?.classList?.remove('error-box');
            this.toasterService.clearAllToaster();
```

code rabbit suggestion please check if correct add this

Reply:
Remove